modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/gmos/GmosHeader.scala

```diff
@@ -52,11 +52,14 @@ final case class GmosHeader(hs: DhsClient, gmosObsReader: GmosHeader.ObsKeywords
         buildInt32(rv.ySize, s"DETRO${i}YS")
       )
   }.toList
+  
+  private val InBeam: Int = 0
```

Comment:
Why not a boolean? is it an `Int` in epics?

Reply:
Yes, it is an Int in EPICS.